sw/deployer/Cargo.toml

```diff
@@ -30,6 +30,11 @@ rust-ini = "0.18.0"
 serial2 = "0.2.26"
 serialport = "4.4.0"
 
+[target.'cfg(unix)'.dependencies]
```

Comment:
Is anything preventing this code to work on macOS?

Reply:
From my understanding "unix" does also include macOS? I didn't test it yet though